src/actions/general.js

```diff
@@ -40,12 +42,33 @@ export function resetPing() {
 }
 
 export function getClientConfig() {
-    return bindClientFunc(
-        Client4.getClientConfigOld,
-        GeneralTypes.CLIENT_CONFIG_REQUEST,
-        [GeneralTypes.CLIENT_CONFIG_RECEIVED, GeneralTypes.CLIENT_CONFIG_SUCCESS],
-        GeneralTypes.CLIENT_CONFIG_FAILURE
-    );
+    return async (dispatch, getState) => {
+        dispatch({type: GeneralTypes.CLIENT_CONFIG_REQUEST}, getState);
+
+        let data;
+        try {
+            data = await Client4.getClientConfigOld();
+        } catch (error) {
+            forceLogoutIfNecessary(error, dispatch);
```

Comment:
No, but it doesn't hurt to have anyway. `bindClientFunc` does this anyways right and unless a 401 is received it won't matter